sql/plan.go

```diff
@@ -411,6 +414,11 @@ type planNode interface {
 	// Values().
 	DebugValues() debugValues
 
+	// Start begins the query/statement and initializes what needs to be
+	// initialized (e.g. final type checking of placeholders, first query
+	// plan). Returns an error if initialization fails.
+	Start() *roachpb.Error
```

Comment:
Seems like this could return `error`.

Also, the commit message that this defines a new interface, but it defines a new method.